dev-util/git_delta/git_delta-0.18.3~git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A syntax-highlighting pager for git, diff, grep, and blame output"
+DESCRIPTION="Delta allows extensive flexibility to the layout and styling of diffs"
+HOMEPAGE="https://dandavison.github.io/delta/"
+COPYRIGHT="2020 Dan Davison"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev=acd758f7a08df6c2ac5542a2c5a4034c664a9ed8
+SOURCE_URI="https://github.com/dandavison/delta/archive/$srcGitRev.tar.gz"
+#future?
+#SOURCE_URI="https://github.com/dandavison/delta/archive/refs/tags/0.18.3.tar.gz"
+SOURCE_DIR="delta-$srcGitRev"
+CHECKSUM_SHA256="546a9a7d405c41c9108242da5875036fbcfbf6d5b3b810c183ee0e2f550ec1ce"
```

Comment:
Switch lines, SOURCE_DIR after CHECKSUM_SHA256

Reply:
You could also add a line after CHECKSUM_SHA256 for the downloaded filename which makes it easier to find/read, like:
`SOURCE_FILENAME="delta-$portVersion-$srcGitRev.tar.gz"`

Reply:
Gotcha.